<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>794</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>906</width>
    <height>794</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>906</width>
    <height>794</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WhoIsIn</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="Overview">
       <attribute name="title">
        <string>Overview</string>
       </attribute>
       <widget class="QLabel" name="logo">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>30</y>
          <width>281</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="overview_table">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>531</width>
          <height>661</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">alternate-background-color: #506799;
selection-background-color: #3C4F78;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Checked-In time</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="circle_blue">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>170</y>
          <width>260</width>
          <height>210</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="trainers_in">
        <property name="geometry">
         <rect>
          <x>605</x>
          <y>240</y>
          <width>171</width>
          <height>121</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Yrsa</family>
          <pointsize>50</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>612</x>
          <y>210</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Checked-In Trainers:</string>
        </property>
       </widget>
       <widget class="QLabel" name="circle_blue_2">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>400</y>
          <width>260</width>
          <height>210</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>665</x>
          <y>430</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>TIME:</string>
        </property>
       </widget>
       <widget class="QLabel" name="time">
        <property name="geometry">
         <rect>
          <x>605</x>
          <y>455</y>
          <width>171</width>
          <height>121</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Yrsa</family>
          <pointsize>35</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <zorder>circle_blue</zorder>
       <zorder>logo</zorder>
       <zorder>overview_table</zorder>
       <zorder>trainers_in</zorder>
       <zorder>label</zorder>
       <zorder>circle_blue_2</zorder>
       <zorder>label_2</zorder>
       <zorder>time</zorder>
      </widget>
      <widget class="QWidget" name="Register">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Manger Trainers</string>
       </attribute>
       <attribute name="toolTip">
        <string>Add new trainer</string>
       </attribute>
       <widget class="QLabel" name="logo_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>30</y>
          <width>281</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="data_table">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>551</width>
          <height>481</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">alternate-background-color: #506799;
selection-background-color: #3C4F78;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Username</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Email</string>
         </property>
        </column>
       </widget>
       <widget class="QGroupBox" name="table_controler_groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>500</y>
          <width>551</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>Table contoler</string>
        </property>
        <widget class="QGroupBox" name="get_user_groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>191</width>
           <height>91</height>
          </rect>
         </property>
         <property name="title">
          <string>Get one trainer</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>171</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="get_user_label">
               <property name="text">
                <string>ID  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="get_user_lineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="show_user_pushButton">
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="geometry">
          <rect>
           <x>221</x>
           <y>25</y>
           <width>191</width>
           <height>69</height>
          </rect>
         </property>
         <property name="title">
          <string>Get all trainers</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QPushButton" name="show_all_users">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="geometry">
          <rect>
           <x>221</x>
           <y>100</y>
           <width>191</width>
           <height>91</height>
          </rect>
         </property>
         <property name="title">
          <string>Get all trainers on date</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>171</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="get_trainers_onDate_label">
               <property name="text">
                <string>Date:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="get_trainers_onDate_dateEdit">
               <property name="dateTime">
                <datetime>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                 <year>2021</year>
                 <month>1</month>
                 <day>1</day>
                </datetime>
               </property>
               <property name="displayFormat">
                <string>dd-MM-yyyy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="show_users_onDate_pushButton">
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>25</y>
           <width>191</width>
           <height>69</height>
          </rect>
         </property>
         <property name="title">
          <string>Search in table</string>
         </property>
         <widget class="QLineEdit" name="search_in_table_lineEdit">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>31</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="create_email_pushButton">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>30</y>
           <width>111</width>
           <height>75</height>
          </rect>
         </property>
         <property name="text">
          <string>Corona
warning email</string>
         </property>
        </widget>
        <widget class="QPushButton" name="reste_datatable_pushButton">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>116</y>
           <width>111</width>
           <height>75</height>
          </rect>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </widget>
       <widget class="QCheckBox" name="selectAll_checkBox">
        <property name="geometry">
         <rect>
          <x>45</x>
          <y>12</y>
          <width>16</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>150</y>
          <width>249</width>
          <height>551</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Register New Trainer</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_RFID_ID_3">
                <property name="text">
                 <string>Username </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_trainer_username"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_trainer_name">
                <property name="text">
                 <string>Name         </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_trainer_name"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_trainer_email">
                <property name="text">
                 <string>E-mail        </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_trainer_email"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_RFID_ID">
                <property name="text">
                 <string>ID                </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_RFID_ID"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_register">
              <property name="text">
               <string>Register</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Delete Trainer</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_RFID_ID_2">
                <property name="text">
                 <string>ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_RFID_ID_detete"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_delete_user">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Set/Get max cheked-in</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="set_max_label">
                  <property name="text">
                   <string>Set Max: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="set_max_lineEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="set_max_pushButton">
                <property name="text">
                 <string>Set</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="get_max_label">
                  <property name="text">
                   <string>Get Max: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>10</width>
                    <height>10</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="view_max_label">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: white;
border-color: black;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="get_max_pushButton">
                <property name="text">
                 <string>Get</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
